code/index.html

```diff
@@ -1,30 +1,88 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Project Business Website</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <header>
+    <div class="hero-image">
+      <div class="hero-text">
+        <h1>I am Felix </h1>
+        <h2>I'm a Cat Whisperer</h2>
+      </div>
+    </div>
+
+  </header>
+  <!-- Signup form -->
+
+  <section class="main">
+
+    <div class="form">
+      <form action="https://httpbin.org/anything" method="POST" target="_blank">
+        <h3>Contact me for a consultation:</h3>
+        <br>
+        <label>
+          Name:<br>
```

Comment:
Here you actually could to the same styling with css instead of using the `<br>` tag! The `<br>` element is from the beginning only meant to be used in texts